client/packages/invoices/src/Prescriptions/LineEditView/hooks/useDraftPrescriptionLines.tsx

```diff
@@ -90,30 +85,15 @@ export const useDraftPrescriptionLines = (
       .filter(stockLine => !stockLine.location?.onHold)
       .sort(SortUtils.byExpiryAsc);
 
-    if (status === InvoiceNodeStatus.New) {
-      let placeholder = lines?.find(
-        ({ type }) => type === InvoiceLineNodeType.UnallocatedStock
-      );
-      if (!placeholder) {
-        placeholder = draftLines.find(
-          ({ type }) => type === InvoiceLineNodeType.UnallocatedStock
-        );
-      }
-      if (placeholder) {
-        const placeholderItem = lines?.find(l => l.item.id === item.id)?.item;
-        if (!!placeholderItem) placeholder.item = placeholderItem;
-        rows.push(
-          createDraftPrescriptionLine({
-            invoiceId: invoiceId ?? '',
-            invoiceLine: placeholder,
-            invoiceStatus: status,
-          })
-        );
-      } else {
-        // Commented out for now until placeholders are implemented for
-        // prescriptions
-        // rows.push(createStockOutPlaceholderRow(invoiceId, item.id));
-      }
+    const allStockLinesExpired = stockLines.every(
+      stockLine =>
+        stockLine.expiryDate && DateUtils.isExpired(stockLine.expiryDate)
+    );
```

Comment:
So, no longer any different logic in new vs picked. Agree we need to confirm if it should be picked if only a placeholder row, but... future problem 😁 Should now be able to set prescribed quantities for all expired stock lines

Reply:
Also doesn't show in detail UI that that it's placeholder either...